tests/unit/Makefile.am

```diff
@@ -188,6 +188,7 @@ if MACOSX
 XFAIL_TESTS = set_domainname_test
 XFAIL_TESTS += process_test
 XFAIL_TESTS += mon_processes_test
+XFAIL_TESTS += rlist_test
```

Comment:
can we put a reference to the ticket here? would be nice if we already did that previously so we have context on WHY these tests are allowed to fail.